src/app/features/date-range/components/date-range/date-range.component.html

```diff
@@ -0,0 +1,27 @@
+<section class="date-range">
+  <div class="date-range-header">
+    <h2 class="date-range-title">{{ dateRangeData.headerText }}</h2>
+    <mat-icon [mat-dialog-close]="false">close</mat-icon>
+  </div>
+  <div class="date-range-calendar">
+    <div class="date-range-calendar-card">
+      <div class="period">
+        <span>{{ selectedRangeValue?.start | date: 'dd.MM.yyyy' }} - </span>
+        <span>{{ selectedRangeValue?.end | date: 'dd.MM.yyyy' }}</span>
+      </div>
+      <mat-calendar [dateFilter]="isDateUnavailable"
+        (selectedChange)="selectedDateRange($event)"
+        [minDate]="minDate" [maxDate]="maxDate" [selected]="selectedRangeValue"
+      >
+      </mat-calendar>
+    </div>
+  </div>
+  <div class="date-range-button">
+    <button mat-raised-button color="primary" class="button-confirm"
+      [disabled]="!selectedRangeValue?.start || !selectedRangeValue?.end"
```

Comment:
designers do not like when we disable buttons. let it be always enabled. if we do not chose a period it emits null.

Reply:
done